yarn.lock

```diff
@@ -1987,6 +1987,12 @@
     classnames "^2.2.6"
     react-media "^1.10.0"
 
+"@gnosis.pm/safe-react-components@git+https://github.com/gnosis/safe-react-components#1d732d38f1f7a6497b404339f6b75f9aeda6da39":
+  version "0.8.0"
+  resolved "git+https://github.com/gnosis/safe-react-components#1d732d38f1f7a6497b404339f6b75f9aeda6da39"
```

Comment:
There are some inconsistencies in the yarn.lock this is still pointing to a github commit.

Maybe running `yarn install` is still needed to grant the sync

Reply:
yep 🙃

Reply:
fixed